pkg/controller/kubelet-config/kubelet_config_nodes.go

```diff
@@ -278,6 +358,15 @@ func RunNodeConfigBootstrap(templateDir string, fgHandler ctrlcommon.FeatureGate
 
 	configs := []*mcfgv1.MachineConfig{}
 
+	// Create auto-sizing MachineConfigs for each pool
+	for _, pool := range mcpPools {
+		autoSizingMC, err := newAutoSizingMachineConfig(pool)
```

Comment:
This is not the place per [this](https://github.com/openshift/machine-config-operator/pull/5387/files#r2517233985)